Raw docstring not ignored when computing similarities

Raw docstrings are not ignored with the `ignore-docstring=yes` in the similarities checker. I think it should be ignored because raw docstrings are also considered a docstring according to [PEP 257](https://www.python.org/dev/peps/pep-0257/)
### Steps to reproduce
Run pylint on the following two files:
`pylint ./one.py ./two.py`
```python
"""One module"""


def one_func():
    r"""hello

    This docstring is duplicated.
    This docstring is duplicated.
    This docstring is duplicated.
    This docstring is duplicated.
    """
    return 1
```
```python
"""Another module"""


def another_func():
    r"""hello

    This docstring is duplicated.
    This docstring is duplicated.
    This docstring is duplicated.
    This docstring is duplicated.
    """
```

### Current behavior
Pylint reports:
```
************* Module temp.pylint_test.two
two.py:1:0: R0801: Similar lines in 2 files
==temp.pylint_test.one:4
==temp.pylint_test.two:4
    r"""hello

    This docstring is duplicated.
    This docstring is duplicated.
    This docstring is duplicated.
    This docstring is duplicated.
    """ (duplicate-code)

------------------------------------------------------------------
Your code has been rated at 6.67/10 (previous run: 6.67/10, +0.00)
```

### Expected behavior
Since the docstrings are ignored by default, raw docstrings should also be ignored: 
```
--------------------------------------------------------------------
Your code has been rated at 10.00/10 (previous run: 10.00/10, +0.00)
```
### pylint --version output
```
pylint 2.3.1
astroid 2.2.5
Python 3.6.8 |Anaconda, Inc.| (default, Dec 30 2018, 01:22:34) 
[GCC 7.3.0]
```

--- a/CONTRIBUTORS.txt
+++ b/CONTRIBUTORS.txt
@@ -290,3 +290,5 @@ contributors:
     - Made C0412 (ungrouped import) compatible with isort
 
 * Nathan Marrow
+
+* Taewon Kim : contributor

--- a/ChangeLog
+++ b/ChangeLog
@@ -7,6 +7,8 @@ What's New in Pylint 2.4.0?
 
 Release date: TBA
 
+* Ignore raw docstrings when running Similarities checker with `ignore-docstrings=yes` option
+
 * Fix crash when calling ``inherit_from_std_ex`` on a class which is its own ancestor
 
   Close #2680

--- a/pylint/checkers/similar.py
+++ b/pylint/checkers/similar.py
@@ -176,7 +176,9 @@ def stripped_lines(lines, ignore_comments, ignore_docstrings, ignore_imports):
     for lineno, line in enumerate(lines, start=1):
         line = line.strip()
         if ignore_docstrings:
-            if not docstring and (line.startswith('"""') or line.startswith("'''")):
+            if not docstring and any(
+                line.startswith(i) for i in ['"""', "'''", 'r"""', "r'''"]
+            ):
                 docstring = line[:3]
                 line = line[3:]
             if docstring:

--- a/pylint/test/input/similar1
+++ b/pylint/test/input/similar1
@@ -20,3 +20,11 @@ fifteen
 sixteen
 seventeen
 eighteen
+
+
+
+
+nineteen
+r"""
+twenty
+"""

--- a/pylint/test/input/similar2
+++ b/pylint/test/input/similar2
@@ -20,3 +20,11 @@ FIFTEEN
 sixteen
 seventeen
 eighteen
+
+
+
+
+NINETEEN
+r"""
+twenty
+"""

--- a/pylint/test/unittest_checker_similar.py
+++ b/pylint/test/unittest_checker_similar.py
@@ -48,7 +48,7 @@ def test_ignore_comments():
    eight
    nine
    ''' ten
-TOTAL lines=44 duplicates=10 percent=22.73
+TOTAL lines=60 duplicates=10 percent=16.67
 """
             % (SIMILAR1, SIMILAR2)
         ).strip()
@@ -84,7 +84,7 @@ def test_ignore_docsrings():
    three
    four
    five
-TOTAL lines=44 duplicates=13 percent=29.55
+TOTAL lines=60 duplicates=13 percent=21.67
 """
             % ((SIMILAR1, SIMILAR2) * 2)
         ).strip()
@@ -99,7 +99,7 @@ def test_ignore_imports():
     assert (
         output.getvalue().strip()
         == """
-TOTAL lines=44 duplicates=0 percent=0.00
+TOTAL lines=60 duplicates=0 percent=0.00
 """.strip()
     )
 
@@ -169,7 +169,7 @@ def test_ignore_nothing():
    three
    four
    five
-TOTAL lines=44 duplicates=5 percent=11.36
+TOTAL lines=60 duplicates=5 percent=8.33
 """
             % (SIMILAR1, SIMILAR2)
         ).strip()

